test/StreamManagerTest.js

```diff
@@ -0,0 +1,769 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+const { time, loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
+const { getSignersAndDeployContracts } = require("./fixtures");
+
+describe.only("StreamManager:", async () => {
+	const amount = 1000
+    const rate = 1500
+    const terminationPeriod = 18 * 24 * 3600; // 18 days
+    const cliffPeriod = 24 * 3600; // 24 hrs
+    const zero = ethers.constants.AddressZero 
+
+	it("Checking fixtures", async () => {
+		await loadFixture(getSignersAndDeployContracts)
+
+	    // Returning the `decimals` for coverage*
+	    const { mockUSDT } = await loadFixture(getSignersAndDeployContracts)
+	    expect(await mockUSDT.decimals()).to.eq(6)
```

Comment:
@CTAPCKPIM The idea here is the following..

Fixture 1 (deploy the contracts)
Fixture 2 (creating something)
Fixture 3 (removing something)

if you need to use Fixture 3, it should contain the first 2, and so on.

In your case I see here duplication.